README.md

```diff
@@ -36,11 +36,24 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
 
 To contribute new features and changes to the website, you would want to run the app locally. Please follow these steps:
 
-1. Clone the project locally. Make sure you have a recent version of Node.js installed on your computer.
 
-2. You can use the included [generated.json](generated.json) as dummy data or you can run `npm run prebuild` to fetch the latest data from GitHub yourself: for this, you will need to set the `GH_PERSONAL_ACCESS_TOKEN` environment variable to a valid GitHub Personal Access Token (PAT). Notice: repositories not maching the criteria listed above (see rules in [generated.json](generated.json)) are automatically removed from [firstissue.json](firstissue.json) when the [generated.json](generated.json) script runs.
+## How to setup the project locally
+1. Fork the repository. To do this, navigate to the repository for your contribution and click on the “Fork” button in the top-right corner.
```

Comment:
Nice, thanks for the update! Two things:

- Step 3 is missing
- I would merge points 1, 2,4 along the line of "Fork and Clone the repository. Then create a new branch. You can do so by: blablabla and then `git clone ...` and then `git checkout ...`"